packages/server/worker/src/lib/runner/process/engine-process-manager.ts

```diff
@@ -237,7 +237,7 @@ function getFlowVersionId(operation: EngineOperation, type: EngineOperationType)
         case EngineOperationType.EXECUTE_FLOW:
             return (operation as ExecuteFlowOperation).flowVersion.id
         case EngineOperationType.EXECUTE_PROPERTY:
-            return (operation as ExecutePropsOptions).flowVersion.id
+            return (operation as ExecutePropsOptions).flowVersion?.id ?? DEFAULT_MCP_DATA.flowVersionId
```

Comment:
I see the function already accept undefined as answer? why not just returns undefined instead of random  string that doesn't represent anything